webapp/src/Service/ScoreboardService.php

```diff
@@ -386,21 +415,16 @@ public function calculateScoreRow(
                 continue;
             }
 
+            // STEP 3:
             $absSubmitTime = (float)$submission->getSubmittime();
             $submitTime    = $contest->getContestTime($absSubmitTime);
 
-            // if correct, don't look at any more submissions after this one.
             if ($judging->getResult() == Judging::RESULT_CORRECT) {
                 $correctJury = true;
                 $timeJury    = $submitTime;
```

Comment:
This should not get updated anymore once set, as otherwise it would track the latest submission time, not the first.

Reply:
It will not be updated again because of the `continue` in line 415.